### PR TITLE
Move E2E into it's own node

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] DX

## Description
Extracting E2E specs into its own build node to speed up CI

## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
Please kindly visit the Travis build to witness the changes. 

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
n/a

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -71,14 +71,17 @@ jobs:
...
       script:
         - yarn lint:frontend
         - yarn test --colors
         - bundle exec rails db:create db:schema:load assets:precompile
         - bundle exec bundle-audit check --update
         - bin/test-console-check
         - yarn build-storybook
+    - stage: test
+      name: "E2E"
+      script:
```
Comment (djuber) [inline .travis.yml:84]: fairly certain you'll want/need the rails db:schema:load and assets:precompile before running e2e here

Comment (maestromac) [reply .travis.yml:84]: Ah right. I'm facing another issue where Travis refuse to start builds

File: .travis.yml [hunk 2]
```diff
@@ -71,17 +71,19 @@ jobs:
...
-      name: "Front-end tests, Storybook, bundle audit, console check, e2e"
+      name: "Front-end tests, Storybook, bundle audit, console check"
       script:
         - yarn lint:frontend
         - yarn test --colors
-        - bundle exec rails db:create db:schema:load assets:precompile
         - bundle exec bundle-audit check --update
         - bin/test-console-check
         - yarn build-storybook
+    - stage: test
```
Comment (maestromac) [inline .travis.yml:81]:
```suggestion
    - stage: Test
```

File: .travis.yml [hunk 3]
```diff
@@ -71,17 +71,19 @@ jobs:
...
+      name: "Front-end tests, Storybook, bundle audit, console check"
       script:
         - yarn lint:frontend
         - yarn test --colors
-        - bundle exec rails db:create db:schema:load assets:precompile
         - bundle exec bundle-audit check --update
         - bin/test-console-check
         - yarn build-storybook
+    - stage: Test
+      name: "E2E"
```
Comment (djuber) [inline .travis.yml:82]: well-that-worked

**Review by djuber — APPROVED** [review]

**Review by nickytonline — APPROVED** [review]
🚢

**Pull request merged**